web/packages/teleport/src/Notifications/Notifications.tsx

```diff
@@ -76,11 +104,33 @@ export function Notifications({ iconSize = 24 }: { iconSize?: number }) {
     ),
   });
 
+  function updateLocalNotificationsState() {
+    const localNotifs = store.getNotifications();
+    const notificationStates = store.getNotificationStates();
+
+    setLocalNotifications(
+      accessListStoreNotificationsToNotifications(
+        localNotifs,
+        notificationStates
+      )
+    );
+  }
+
   // Fetch first page on first render.
   useEffect(() => {
     fetch();
+
+    store.subscribe(updateLocalNotificationsState);
+
+    return () => {
+      store.unsubscribe(updateLocalNotificationsState);
```

Comment:
i don't think this is necessary 🤔 

the `useStore(ctx.storeNotifications);` already subscribes to the store state changes (ctx.storeNotification contains `state`)

so i think the correct way to use this then is 

instead of this `localNotifications` as a react state

```ts
const [localNotifications, setLocalNotifications] = useState<
    NotificationType[]
  >([]);
```

we should do this instead:

```ts
const localNotifications = React.useMemo(
    () =>
      accessListStoreNotificationsToNotifications(
        ctx.storeNotifications.getNotifications(),
        ctx.storeNotifications.getNotificationStates()
      ),
    [ctx.storeNotifications.state]
  );
```

and we dont need to subscribe to `updateLocalNotificationsState`